src/ledger/LedgerTxnEntry.cpp

```diff
@@ -100,13 +101,27 @@ LedgerTxnEntry::operator=(LedgerTxnEntry&& other)
         LedgerTxnEntry otherCopy(other.mImpl.lock());
         swap(otherCopy);
         other.mImpl.reset();
+        mIsInitialized = true;
```

Comment:
Same question about initializing with something that is not initialized.

Reply:
Updated